Lint all files in a directory by expanding arguments (#5682)
PyCQA/pylint: doc/user_guide/run.rst

file or it is an implicit namespace package) or if "directory" is in the
python path.

> By default, pylint will exits an error when one of the arguments is a directory which is not

Reviewer: ```suggestion
By default, pylint will exit with an error when one of the arguments is a directory which is not
```

> By default, pylint will result with an error when one of the arguments is directory which is not

Reviewer: ```suggestion
By default, pylint will exits an error when one of the arguments is a directory which is not
```

> a python package. In order to run pylint over all modules and packages within file system

Reviewer: ```suggestion
a python package. In order to run pylint over all modules and packages within the provided
```

> subtree of directory, ``--recursive=y`` option must be provided.

Reviewer: ```suggestion
subtree of a directory, the ``--recursive=y`` option must be provided.
```
